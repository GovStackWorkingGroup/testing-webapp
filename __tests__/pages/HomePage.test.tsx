import HomePage from '../../pages/index'
import { render } from '../test-utils';

describe('Unit test for HomePage', () => {
  it('renders the home page', () => {
    const { container } = render(
      <HomePage />
      )

    expect(container).toMatchSnapshot();
  })
})
