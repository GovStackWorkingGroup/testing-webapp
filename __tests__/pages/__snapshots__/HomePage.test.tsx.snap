// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unit test for HomePage render the Home page 1`] = `
<div>
  <div>
    <main>
      <div
        class="table"
      >
        <div
          class="table-results-count"
        >
          <p>
            0
             results
          </p>
        </div>
        <div
          class="table-body"
        >
          <div
            class="product-table-header"
          >
            <div
              class="empty-space"
            />
            <div
              class="product-table-content"
            >
              <div>
                <p>
                  Product Name
                </p>
              </div>
              <div>
                <p>
                  Building Blocks
                </p>
              </div>
              <div>
                <p>
                  Last Update
                </p>
              </div>
              <div>
                <p>
                  Overall Compatibility
                </p>
              </div>
            </div>
          </div>
          <div
            class="product-table-row"
          >
            <div
              class="details-arrow"
            />
            <div
              class="product-table-content"
            >
              <div>
                <p />
              </div>
              <div>
                <p />
              </div>
              <div>
                <p />
              </div>
              <div>
                <p />
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>
</div>
`;
